cmd/earthly/main.go

```diff
@@ -2629,7 +2629,20 @@ func (app *earthlyApp) actionBuildImp(c *cli.Context, flagArgs, nonFlagArgs []st
 	cleanCollection := cleanup.NewCollection()
 	defer cleanCollection.Close()
 
-	go terminal.ConnectTerm(c.Context, app.debuggerHost)
+	go func() {
+		// Dialing doesnt accept URLs, it accepts an address and a "network". These cannot be handled as URL schemes.
+		// Since Shellrepeater hard-codes TCP, we drop it here and log the error if we fail to connect.
+
+		u, err := url.Parse(app.debuggerHost)
+		if err != nil {
+			panic("debugger host was not a URL")
```

Comment:
This `panic` is consistent with what we do if parsing fails when starting buildkit.

Reply:
should we also error out if `u.Scheme != "tcp"`?

Reply:
We already do. See the URL validation we do in `before`: https://github.com/earthly/earthly/blob/b3f056b510d822731710f3e463abbc27ed19fcd7/cmd/earthly/main.go#L1111-L1126

Reply:
awesome! thanks for pointing that out.